src/mbgl/gl/attribute.hpp

```diff
@@ -255,8 +266,9 @@ class Attributes {
     }
 
     template <class DrawMode>
-    static Bindings bindings(const VertexBuffer<Vertex, DrawMode>& buffer) {
```

Comment:
🤔 @jfirebaugh good question — I added it after [this build](https://circleci.com/gh/mapbox/mapbox-gl-native/6870?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link), which was terminated during the render tests with
```
terminate called after throwing an instance of 'mbgl::gl::Error'
  what():  glEnableVertexAttribArray(location): Error GL_INVALID_VALUE at ../../../src/mbgl/gl/attribute.cpp:62
```
([glEnableVertexAttribArray documentation](https://www.khronos.org/registry/OpenGL-Refpages/es2.0/xhtml/glEnableVertexAttribArray.xml))
But I was able to reproduce that error locally at the time using a fill-color property function, which now works locally if I remove it… I'll remove it and see if any of the CI builds still have issues.